akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -512,6 +546,11 @@ class Http2ServerSpec extends AkkaSpec with WithInPendingUntilFixed {
     def expectRequestRaw(): HttpRequest = requestIn.requestNext() // TODO, make it so that internal headers are not listed in `headers` etc?
     def emitResponse(streamId: Int, response: HttpResponse): Unit =
       responseOut.sendNext(response.addHeader(Http2StreamIdHeader(streamId)))
+
+    def expectGracefulCompletion(): Unit = {
+      toNet.expectComplete()
```

Comment:
is there a better way to check the test probe's for completion/close, maybe I address that in the connection close handle pr..?